adapters/tlsio_openssl.c

```diff
@@ -1607,6 +1619,21 @@ static STACK_OF(X509_CRL) *crls_http_cb(X509_STORE_CTX *ctx, X509_NAME *nm)
     return crls;
 }
 
+static int allow_CRL_fetch_error(int status, X509_STORE_CTX *ctx)
+{
+    if(X509_V_ERR_UNABLE_TO_GET_CRL == X509_STORE_CTX_get_error(ctx))
+    {
+        LogInfo("Ignoring CRL Downlaod failure\n");
+        return 1;
+    }
+    else
+    {
+        LogError("Error %d was unexpected\n", X509_STORE_CTX_get_error(ctx));
+        return status;
```

Comment:
Could the value of ```status``` also be set to 1?

Reply:
Could.

From the OpenSSL Docs:

> The ok parameter to the callback indicates the value the callback should return to retain the default behaviour. If it is zero then an error condition is indicated. If it is 1 then no error occurred.

https://www.openssl.org/docs/man1.1.0/man3/X509_STORE_CTX_set_verify_cb.html

I only want to change the behavior for the CRL get operation, so I return the default behavior code for the rest.

Reply:
Got it.

I also missed this earlier but you have typo at line 1626: ```Downlaod``` instead of ```Download```